tests/misc_env/install_prereq.py

```diff
@@ -58,6 +58,8 @@ def install_prereq(ceph, timeout=1800, skip_subscription=False, repo=False, rhbu
         # TODO enable only python3 rpms on both rhel7 &rhel8 once all component suites(rhcs3,4) are comptatible
         if distro_ver.startswith('8'):
             rpm_all_packages = ' '.join(rpm_packages.get('py3'))
+            if str(rhbuild).startswith('5'):
+                rpm_all_packages = rpm_all_packages + 'lvm2' + 'podman'
```

Comment:
If the `cephadm` package requires those two packages in order to function properly, but Yum does not automatically pull those in, that is a bug in my opinion.

Reply:
@ktdreyer Upstream doc says that  
> Dependencies are installed automatically by the bootstrap process below.
We will verify whether this behavior is happening and update the script accordingly and also file RFE if required.

Reply:
Thanks. I verified today that the latest cephadm RPM package has "Requires: lvm2" and "Recommends: podman". Maybe we should change that to "Requires: podman" now?